Siri/L6/Level6.java

```diff
@@ -0,0 +1,113 @@
+                            + t.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+                } catch (Exception e) {
+                    System.out.println(line + e.toString() + "\n" + line);
+                }
+            } else if (word.equals("deadline")) {
+                //creates deadline task with "[D]" prefix and deadline postfix and adds to list
+                while (sc.hasNextLine()) {
+                    String[] userInput = sc.nextLine().split("/by");
+                    // The userInput array now contains [taskDescription, deadline]
+                    Deadline deadLineTask = new Deadline(userInput[0], userInput[1]);
+                    arrList.add(deadLineTask);
+                    System.out.println(line + "     This task has been added to your list:\n       "
+                            + deadLineTask.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+                    break;
+                }
+            } else if (word.equals("event")) {
+                //creates event task with "[E]" prefix and event date postfix and adds to list
+                while (sc.hasNextLine()) {
+                    String[] userInput = sc.nextLine().split("/at");
+                    // The userInput array now contains [taskDescription, event date]
+                    Event eventTask = new Event(userInput[0], userInput[1]);
+                    arrList.add(eventTask);
+                    System.out.println(line + "     This task has been added to your list:\n       "
+                            + eventTask.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+                    break;
+                }
+            } else if (word.equals("delete")) {
+                    int listIndex = sc.nextInt() - 1;
+                    Task tempT = arrList.get(listIndex);
+                    arrList.remove(listIndex);
+                    System.out.println(line + "     Sure. I've removed this task from the list:\n       "
+                            + tempT.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+            } else {
+                System.out.println(line + "     Please enter a valid task type with description.       \n" + line);
+            }
+        }
+    }
+
+    static void checkRestOfInput(String description) throws EmptyDescException {
+
+        if (description.isBlank()) {
+            throw new EmptyDescException("     The description of a todo task cannot be empty.");
+        } else {
+            return;
+        }
+    }
+}
```

Comment:
Just a general suggestion, maybe you can also consider modifying your code structure? I think you can take a look at others' code, the code is structured within the src/java directory, and no need to have a separate folder for different levels as it is handled by git/version control.